### PR TITLE
feat (dashboard): don't show deprecated plans

### PR DESCRIPTION
**Pull request by dbarrosop**

### CODE REVIEW

**Review by NesManrique — APPROVED** [review]

File: dashboard/src/features/projects/overview/components/OverviewTopBar/OverviewTopBar.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ export default function OverviewTopBar() {
               <div className="mt-1 inline-grid grid-flow-col items-center justify-start gap-2 md:mt-0">
                 <Chip
                   size="small"
-                  label={isPro ? 'Pro' : 'Starter'}
+                  label={currentProject.plan.name}
                   color={isPro ? 'primary' : 'default'}
```
Comment (onehassan) [inline dashboard/src/features/projects/overview/components/OverviewTopBar/OverviewTopBar.tsx:89]: plan is `null` when running the local dashboard. I suggest doing optional chaining `currentProject.plan?.name` and maybe add a default

File: dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ function AllWorkspaceApps() {
 
                 <Chip
                   size="small"
-                  label={project.plan.isFree ? 'Starter' : 'Pro'}
+                  label={project.plan.name}
```
Comment (onehassan) [inline dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx:83]: same here

File: dashboard/src/gql/app/prefetchNewApp.graphql [hunk 3]
```diff
@@ -32,7 +32,7 @@ query PrefetchNewApp {
   regions(order_by: { city: asc }) {
     ...PrefetchNewAppRegions
   }
-  plans(order_by: { sort: asc }) {
+  plans(order_by: {sort: asc}, where: {deprecated: {_eq: false}}) {
     ...PrefetchNewAppPlans
```
Comment (onehassan) [inline dashboard/src/gql/app/prefetchNewApp.graphql:36]: I think there's a formatting issue here as there should be a space after `{`

File: .changeset/sour-fishes-talk.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@nhost/dashboard': minor
+---
+
+don't show deprecated plans
```
Comment (onehassan) [inline .changeset/sour-fishes-talk.md:5]: I think it's better to add a `chore: ` or `feat: ` to this so it's consistent with the rest of the changelog

File: dashboard/src/features/projects/overview/components/OverviewTopBar/OverviewTopBar.tsx (hunk 1 above)
Comment (dbarrosop) [reply dashboard/src/features/projects/overview/components/OverviewTopBar/OverviewTopBar.tsx:89]: this isn't rendered on the local page though

File: dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx (hunk 2 above)
Comment (dbarrosop) [reply dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx:83]: the local dashboard doesn't have access to this page

File: dashboard/src/gql/app/prefetchNewApp.graphql (hunk 3 above)
Comment (dbarrosop) [reply dashboard/src/gql/app/prefetchNewApp.graphql:36]: I am not sure I follow, it follows the same format as with `{sort: asc}`, for instance. Do we have a linter?

File: .changeset/sour-fishes-talk.md (hunk 4 above)
Comment (dbarrosop) [reply .changeset/sour-fishes-talk.md]: fixed

File: dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx (hunk 2 above)
Comment (onehassan) [reply dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx:83]: 👍

**Review by onehassan — APPROVED** [review]

**Pull request merged**